Providers/nxOMSAutomationWorker/automationworker/3.x/worker/main.py

```diff
@@ -67,7 +67,7 @@ def get_worker_and_sandbox_processes(process_list):
         """
         worker_and_sandbox_processes = []
         for process in process_list:
-            if ("python" in process.cmd or "python3" in process.cmd) and ("hybridworker.py" in process.cmd or "sandbox.py" in process.cmd) \
```

Comment:
this is a python3 script right ?

Reply:
for this release we will also have to consider "python" because the old version of our module would have started the process as "python <hybridworker.py>", when the version updates this will not stop the processes started by the older version.

Reply:
Is this a continuously running process? When is it stopped?

Yes, this is python3 script. After these changes checking for python also didn't make much sense here. So, changed it to python2 in order to highlight and discuss this.

Reply:
Changes are pushed as per discussion